database/json_file_storage.go

```diff
@@ -34,30 +34,30 @@ type JSONFileStorage struct {
 }
 
 // NewJSONFileStorage creates a new JSONFileStorage instance
-func NewJSONFileStorage(logger logging.Logger, dir string, networks []ids.ID) (*JSONFileStorage, error) {
+func NewJSONFileStorage(logger logging.Logger, dir string, chainIDs []ids.ID) (*JSONFileStorage, error) {
```

Comment:
Nit: Can we name it `blockchainIDs` instead of `chainIDs` to avoid possible confusion?